pkg/server/grpc_server.go

```diff
@@ -312,9 +295,108 @@ func (s *server) ListPath(r *api.ListPathRequest, stream api.GoBgpService_ListPa
 	return send()
 }
 
+func (s *server) watchEvent(ctx context.Context, r *api.WatchEventRequest, fn func(*api.WatchEventResponse, time.Time)) error {
+	if r == nil {
+		return fmt.Errorf("nil request")
+	}
```

Comment:
`watchEvent()` should return gRPC errors?
https://pkg.go.dev/google.golang.org/grpc/codes